myip.go

```diff
@@ -11,6 +11,7 @@ import (
 
 const (
 	cloudflare = "http://1.1.1.1/cdn-cgi/trace"
+	ifconfig.me = "http://ifconfig.me"
```

Comment:
Consider renaming this to ifconfigMe. Generally you want to avoid non-alphanumeric characters.

Reply:
Done.